src/main/webapp/assets/modules/common/challenge/submissionChallenge.js

```diff
@@ -1,4 +1,4 @@
-techlooper.directive("submissionChallenge", function (localStorageService, apiService, $timeout, $rootScope, $location) {
+techlooper.directive("submissionChallenge", function (localStorageService, apiService, $timeout, $rootScope, $location, utils, jsonValue) {
```

Comment:
Line is too long.<br>'techlooper' is not defined.<br>'jsonValue' is defined but never used.<br>'utils' is defined but never used.